packages/airnode-node/src/logger.ts

```diff
@@ -1,4 +1,5 @@
-import { formatDateTimeMs } from './utils/date-utils';
+import { logger as loggerUtils } from '@api3/airnode-utilities';
```

Comment:
Is this aliased as `loggerUtils` to reduce confusion with this file name? `logger` doesn't seem to be used anywhere else in this file